raft/src/main/java/org/apache/kafka/raft/QuorumStateStore.java

```diff
@@ -18,14 +18,23 @@
 
 import java.io.IOException;
 
-public interface ElectionStore {
+/**
+ *  Maintain the save and retrieval of quorum state information, so far only supports
+ *  read and write of election states.
+ *
+ * TODO 1: add clusterId support
+ * TODO 2: add reassignment support
+ * TODO 3: add applied offset support. We do not need this before reassignment is implemented
+ * because a voter depends on BeginQuorumEpoch to learn the latest epoch, not from the LeaderChangeMessage.
+ */
+public interface QuorumStateStore {
 
     /**
      * Read the latest election state.
      * @return The latest written election state or `ElectionState.withUnknownLeader(0)` if there is none.
      * @throws IOException For any error encountered reading from the storage
      */
-    ElectionState read() throws IOException;
+    ElectionState readElectionState() throws IOException;
```

Comment:
nit: a little annoying to have the naming inconsistency with `write`. Should we make that `writeElectionState`?

Reply:
Sounds good